src/core/handle.ml

```diff
@@ -462,15 +468,15 @@ let handle_cmd : sig_state -> p_command ->
             (* Make sure the operator has a fully qualified [qid]. *)
             let unop = (s, prio, with_path sym.sym_path qid) in
             out 3 "(conf) %a %a\n" pp_symbol sym pp_hint (Prefix unop);
-            add_unop ss s (sym, unop)
+            add_unop ss (Pos.none s) (sym, unop)
```

Comment:
Use cmd.pos instead ? Idem for binop below.

Reply:
I was rather thinking of modifying the unop and binop types to include the position of the symbol declaration, and have `unop = string loc * priority * qident`